### ISSUE TITLE
Feature Request: Marketplace announcements in auction chat.

### ISSUE THREAD

**Issue by zdxn** [zdxn/ValkyriesFall#4]
Issue:
Currently nothing happens when players post market place listings.

Resolution:
Every so often a message should be posted in Auction chat that list the number of auctions in the various categories. As an addition players could toggle a feature when listing a new item to post into the auction chat a brief description. 
Ex: "Anubis has placed a Shadow Sword for sale on the market place for 5,000,000 gold. <link to market>"

Urgency:
1

**Comment by zdxn** [comment]
Random auction messages should confirm there isn't a flood of previously posted automated auction messages as well when posting to ensure there isn't spam in off hours or periods when chat is slow.

**Comment by DavePearcey** [comment]
Done.
